prow/config/templates/plugins-ConfigMap.yaml

```diff
@@ -18,19 +18,25 @@ metadata:
   name: plugins
 data:
   plugins.yaml: |
+    triggers:
+    - repos:
+      - {{ .Values.github.organisation }}
+      only_org_members: true
+
     plugins:
       {{ .Values.github.organisation }}:
-      - approve
-      - assign
-      - blunderbuss
-      - cat
-      - dog
-      - help
-      - heart
-      - hold
-      - label
-      - lgtm
-      - trigger
-      - verify-owners
-      - wip
-      - yuks
\ No newline at end of file
+        plugins:
+        - approve
+        - assign
+        - blunderbuss
+        - cat
+        - dog
+        - help
+        - heart
+        - hold
+        - label
+        - lgtm
+        - trigger
+        - verify-owners
+        - wip
+        - yuks
```

Comment:
Could we crawl through these and pull out the silly ones? dog, cat and yuks for example.

Reply:
Agreed they are not the most useful plugins but they are fun 😂  - i vote for keeping them for now